src/test/java/zd/zero/waifu/motivator/plugin/listeners/WaifuUnitTesterImplTest.java

```diff
@@ -1,47 +0,0 @@
-package zd.zero.waifu.motivator.plugin.listeners;
```

Comment:
As we are now using the test topic from Intellij, this test class is now irrelevant.